tests/specs/relay-general-func.spec.ts

```diff
@@ -0,0 +1,110 @@
+import { Page } from '@playwright/test';
+import test, { expect }  from '../fixtures/basePages'
+import { defaultScreenshotOpts } from '../utils/helpers';
+
+// using logged in state outside of describe block will cover state for all tests in file
+test.use({ storageState: 'state.json' })
+test.describe('Free - General Functionalities, Desktop', () => {
+
+    test.beforeEach(async ({ dashboardPage, page }) => {
+      await dashboardPage.open()
+      await checkForSignInButton(page)
+      await dashboardPage.deleteMask()   
+    });    
+  
+    test('Verify that the Header is displayed correctly for a Free user that is logged in, C1812639', async ({ dashboardPage }) => {
+      await expect(dashboardPage.header).toHaveScreenshot(
+        'dashboardHeader.png',
+        defaultScreenshotOpts
+      );
+    })
+  
+    test.fixme('Verify that the "Upgrade" button redirects correctly,  C1812640, 1808503', async ({ dashboardPage, page }) => {
+      await dashboardPage.upgrade()
+      expect(page.url()).toContain('/premium/')
+    })
+  
+    test('Verify that the "Profile" button and its options work correctly, C1812641', async ({ dashboardPage }) => {
+      await dashboardPage.userMenuButton.click()
+      await expect(dashboardPage.userMenuPopUp).toHaveScreenshot(
+        'userMenuPopUp.png',
+        defaultScreenshotOpts
+      );
+      await dashboardPage.userMenuButton.click()
+  
+      await dashboardPage.relayExtensionBanner.scrollIntoViewIfNeeded()
+      await expect(dashboardPage.relayExtensionBanner).toHaveScreenshot(
+        'relayExtensionBanner.png',
```

Comment:
The Chrome version seems to have the user menu overlapping it? Feels like maybe the screenshot of this banner should go into a separate test? (Because it doesn't seem too related to the Profile button.)

Reply:
i dont believe we actually need screenshots from all 3 majors browsers, what do you think about maintaining only webkit(safari) screenshots?

Reply:
As just mentioned, that works for me. (Though it's still suspicious that the menu was still open :) )